docs/blockchain/contracts/nft-contract/nft-actions/mknftofr.a.experimental.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'mknftofr.a'
+order: 37
+
+---
+
+# mknftofr.a
+
+Make an offer on a given token.
+
+## Technical Behavior
+
+The offer should be done at the time which is in the range of trading window of the NFT factory from which the NFT was issued.
+
+The action stores the offer to `nftoffer.a` table with the specified arguments. The new offer ID is read from `next.nftofr` table whose value field is then incremented.
+
+`eosio.nftram` pays RAM usage.
+
+The offered price will be transferred to `eosio.nftofr` account and will be kept until the offer is either accepted or cancelled.
+
+An account will not be able to make offers on the same NFTs that they already made.
+
+An account will not be able to make more offers on NFTs than `max_active_offer_per_user` of `offercfg.a`, which records the global configurations.
```

Comment:
Should be
An account will not be able to make more offers on either factories or tokens ...
or 
Should be
An account will not be able to make more offers on either factories or NFTs ...

Reply:
no update here only changes NFT to Uniq